src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -375,3 +375,31 @@ If you’re seeing errors with the message “Non-Error exception (or promise re
 You can see the contents of the non-Error object in question in the `__serialized__` entry of the “Additional Data” section.
 
 To get better insight into these error events, we recommend finding where the plain object is being passed or thrown to Sentry based on the contents of the `__serialized__` data, and then turning the plain object into an Error object.
+
+## Capturing Resource 404s
+
+By default, Sentry does not capture errors when a resource (like an image or a css file) fails to load. If you would like it to do so, you can use the following code. (Note: We recommend loading Sentry as early as possible in any case, but this method in particular will only work if Sentry is loaded before other resources.)
```

Comment:
```suggestion
By default, Sentry does not capture errors when a resource (like an image or a css file) fails to load. If you would like it to do so, you can use the following code. (Note: We recommend loading Sentry as early as possible in any case, but this specific method will only work if Sentry is loaded before other resources.)
```

Reply:
This one I think I'm going to leave as is, because I feel like it better captures the spirit of "especially here" that I was going for.